_setup.py

```diff
@@ -40,39 +34,16 @@ def install_rexpy(forceInstall):
     print("Error: You're not running setup.py from the root directory. Please run this from the root directory and try again")
     return
 
-  if forceInstall == False and is_rexpy_installed():
-    print(f"rexpy is already installed - skipping install")
-    return
-
-  # rexpy is located in build/scripts/rexpy
-  cwd = os.getcwd()
-  new_wd = os.path.join(cwd, "build", "scripts", "rexpy")
-  os.chdir(new_wd)
-
-  # now run the install script.
-  os.system(f"py {os.path.join(new_wd, 'install.py')} install")
-
-  # this will result in a few new folders generated, which we can safely remove
-  # these folders are "build", "dist" and "rexpy.egg-info"
-  shutil.rmtree("build")
-  shutil.rmtree("dist")
-  shutil.rmtree("rexpy.egg-info")
-
-  # reset the working directory
-  os.chdir(cwd)
-
-if __name__ == "__main__":
-  parser = argparse.ArgumentParser()
-  parser.add_argument("-update_rexpy", help="update rexpy, this is basically a reinstall", action="store_true")
-
-  args, unknown = parser.parse_known_args()
-
-  install_rexpy(args.update_rexpy)
-
-  # now that rexpy is installed, we can safely call the rest of the code
+  __install_externals()
```

Comment:
pip needs to rerun on rexpy if it's downloaded to addd it to the python libraries